src/providerhub/HISTORY.rst

```diff
@@ -3,6 +3,24 @@
 Release History
 ===============
 
+0.2.0
+++++++
+* Support commands for
+`az providerhub notification-registration`
+`az providerhub operation`
+`az providerhub sku`
+
+* Enable creation of nested resource types and adding extension categories.
+
+Refine and update initial release commands.
+* [BREAKING CHANGE] `az providerhub manifest checkin`:
+`--arm-manifest-location` renamed to `--baseline-arm-manifest-location` or `--manifest-location`
+* [BREAKING CHANGE] `az providerhub default-rollout create`:
+`--row2-wait-duration` moved to `--row2 wait-duration`
+`--skip-regions` moved to `--canary skip-regions`
+* [BREAKING CHANGE]: `--provider-type` type changed from enum to string.
+* [BREAKING CHANGE]: `--routing-type` type changed from enum to string.
```

Comment:
Why not add valid enum value for them?

Reply:
Actually users should be able to specify multiple provider types, ie. --provider-type "Internal, Hidden"
The routing type enum seems to be updated somewhat frequently in our code so it may be better for this to be a bit more flexible as a string.

Reply:
That make sense. Please add the available enum value in the help messages for hint.

Reply:
Sounds good, included available values in the help file.